packages/core/src/App/Constants/routes-config.js

```diff
@@ -356,6 +360,12 @@ const getModules = () => {
             is_authenticated: false,
             getTitle: () => localize("Trader's Hub"),
         },
+        {
+            path: routes.callback_page,
+            component: CallbackPage,
+            is_authenticated: false,
+            getTitle: () => localize('Callback'),
```

Comment:
[R] no need to localize this, 

[M] ah, if its possible to get this title, instead of checking window.location.pathname.includes you can also call this function to check if the page is callback